controller/main.go

```diff
@@ -123,14 +123,17 @@ func (c *controller) SetPools(l log.Logger, pools *config.Pools) controllers.Syn
 	level.Debug(l).Log("event", "startUpdate", "msg", "start of config update")
 	defer level.Debug(l).Log("event", "endUpdate", "msg", "end of config update")
 
-	if pools == nil || pools.ByName == nil {
-		level.Error(l).Log("op", "setConfig", "error", "no MetalLB configuration in cluster", "msg", "configuration is missing, MetalLB will not function")
+	if pools == nil || len(pools.ByName) == 0 {
```

Comment:
this is not related to "changing the logging". If there is a reason for this change, please add it as a separate commit with the reasoning behind the change.